MessageStackView/Views/Badges/BadgeContainerView.swift

```diff
@@ -14,6 +14,9 @@ import UIKit
 /// Inherit `ShakeView` to add badge animations
 open class BadgeContainerView: UIView {
     
+    /// Overrideable default shadow components for `BadgeContainerView`
+    static var shadowComponents: UIView.ShadowComponents = .default
```

Comment:
Let's just this in `Constants`.
In a lot of projects I make constants like
```
// MARK: - Constants

extension UIView.ShadowComponents {
    static let shadowComponents: UIView.ShadowComponents = .default
}
```

But for this project I follows the theme of a `Constants` `struct` inside the `class`/`struct`

Reply:
Maybe you have gone this way for overridable reasons. In that case IMO the default should be a constant (as you've done). But the instance should be able to override? Happy either way here!